docs/en/interfaces/cli.md

```diff
@@ -1,63 +1,143 @@
 ---
 slug: /en/interfaces/cli
 sidebar_position: 17
-sidebar_label: Command-Line Client
-title: Command-Line Client
+sidebar_label: ClickHouse Client
+title: ClickHouse Client
 ---
-import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_native.md';
 
-## clickhouse-client
+ClickHouse provides a native command-line client for executing SQL queries directly against a ClickHouse server. It supports both interactive mode (for live query execution) and batch mode (for scripting and automation). Query results can be displayed in the terminal or exported to a file, with support for all ClickHouse output [formats](formats.md), such as Pretty, CSV, JSON, and more.
 
-ClickHouse provides a native command-line client: `clickhouse-client`. The client supports [command-line options](#command-line-options) and [configuration files](#configuration_files).
+The client provides real-time feedback on query execution with a progress bar and the number of rows read, bytes processed and query execution time. It supports both [command-line options](#command-line-options) and [configuration files](#configuration_files).
 
-[Install](../getting-started/install.md) it from the `clickhouse-client` package and run it with the command `clickhouse-client`.
 
-``` bash
-$ clickhouse-client
-ClickHouse client version 20.13.1.5273 (official build).
-Connecting to localhost:9000 as user default.
-Connected to ClickHouse server version 20.13.1.
+## Install
+
+The easiest way to install ClickHouse is with this command:
+
+```bash
+curl https://clickhouse.com/ | sh
+```
+
+See [Install ClickHouse](../getting-started/install.md) for more installation options.
+
+Different client and server versions are compatible with one another, but some features may not be available in older clients. We recommend using the same version for client and server.
+
+
+## Run
+
+:::note
+If you are using the ClickHouse single binary, use `./clickhouse client` instead of `clickhouse-client`.
```

Comment:
or leave a comment here saying this is what will be the case if just using the instructions above with no install

Reply:
done